src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -71,6 +75,30 @@ public void testInitialize() {}
 	@Override
 	public void testExecute() {}
 
+	byte[] matchConfigurationCANData() {
```

Comment:
I think this should be moved to a child of CANInformer - this is a lot of logic that is out of place in Robot.java